keras/backend/common.py

```diff
@@ -28,11 +53,38 @@ def set_epsilon(e):
 def floatx():
     '''Returns the default float type, as a string
     (e.g. 'float16', 'float32', 'float64').
+
+    # Return
+        String, the current default float type
+
+    # Example
+    ```python
+        >>> keras.backend.floatx()
+        'float32'
+    ```
     '''
     return _FLOATX
 
 
 def set_floatx(floatx):
+    '''Sets the default float type.
+
+    # Argument
+        floatx: string. 'float16', 'float32', or 'float64'.
+
+    # Return
+        `None`
+
+    # Example
+    ```python
+        >>> from keras import backend as K
+        >>> K.floatx()
+        'float32'
+        >>> K.set_floatx('float16')
+        >>> K.floatx()
+        'float16'
```

Comment:
This is somewhat problematic because this will be ignored in most Keras functions. But that's a separate problem I suppose, which we will fix in a different PR.